node-fs-writefile/cp.js

```diff
@@ -0,0 +1,9 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+const argv = process.argv.slice(2);
```

Comment:
You might consider destructuring here to make the usage of the arguments more clear:
`const [inFile, outFile] = process.argv.slice(2);`

Reply:
That is a great improvement much better! Thank you